jib-cli/src/main/java/com/google/cloud/tools/jib/cli/jar/JarFiles.java

```diff
@@ -94,4 +80,16 @@ public static JibContainerBuilder toJibContainerBuilder(
 
     return containerBuilder;
   }
+
+  private static String deriveBaseImage(
+      ArtifactProcessor processor, CommonContainerConfigCliOptions cliOptions) {
+    return cliOptions
+        .getFrom()
+        .orElse(
+            processor.getJavaVersion() <= 8
+                ? "eclipse-temurin:8-jre"
+                : processor.getJavaVersion() <= 11
+                    ? "eclipse-temurin:11-jre"
+                    : "azul/zulu-openjdk:17-jre");
```

Comment:
Perhaps we could use a standard if-else block here instead of a ternary operation.